x-pack/plugin/core/src/main/resources/async-search.json

```diff
@@ -0,0 +1,29 @@
+{
+  "index_patterns": [
+    ".async-search-${xpack.async-search.template.version}*"
+  ],
+  "order": 2147483647,
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.auto_expand_replicas": "0-1",
+    "index.lifecycle.name": "async-search",
+    "index.lifecycle.rollover_alias": ".async-search-${xpack.async-search-template.version}",
+    "index.format": 1
+  },
+  "mappings": {
+    "_doc": {
+      "dynamic": false,
```

Comment:
let's make it strict?

Reply:
Sure, I pushed https://github.com/elastic/elasticsearch/commit/9056b873039b5f087a7f98dee1fd31d3850dde11